src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesManagement.Codeunit.al

```diff
@@ -931,11 +971,19 @@ codeunit 396 NoSeriesManagement
     begin
     end;
 
-    [IntegrationEvent(false, false)]
-    internal procedure OnBeforeUpdateNoSeriesLine(var NoSeriesLine: Record "No. Series Line"; NewNo: Code[20]; NewFieldName: Text[100]; var IsHandled: Boolean)
+#if not CLEAN24
+    [Obsolete('This is a temporary method for compatibility only. Please use the "No. Series" codeunit instead', '24.0')]
+    internal procedure RaiseObsoleteOnBeforeUpdateNoSeriesLine(var NoSeriesLine: Record "No. Series Line"; NewNo: Code[20]; NewFieldName: Text[100]; var IsHandled: Boolean)
```

Comment:
```suggestion
    internal procedure RaiseObsoleteOnBeforeUpdateNoSeriesLine(var NoSeriesLine: Record "No. Series Line"; NewNo: Code[20]; NewFieldName: Text[100]; var IsHandled: Boolean)
```
This procedure is internal and hence does not need to be obsoleted.

Reply:
It does not need to be obsoleted but it can be so i'll keep it.